core/src/main/java/com/linecorp/armeria/common/SerializationFormat.java

```diff
@@ -67,10 +75,16 @@
                                    "unknown", create("application", "x-unknown")));
 
         // Load all serialization formats from the providers.
-        ServiceLoader.load(SerializationFormatProvider.class,
-                           SerializationFormatProvider.class.getClassLoader())
-                     .forEach(p -> p.entries().forEach(e -> register(mutableUriTextToFormats,
+        final List<SerializationFormatProvider> providers = Streams.stream(
```

Comment:
ImmutableList.copyOf, think it should work

Reply:
Fixed. Thanks 😉